Python/PlenoInterview/Problem1/order.py

```diff
@@ -0,0 +1,40 @@
+class order:
+
+    def __init__(self, numPeople):
+        self.orderItems = []
+        self.numPeople = numPeople
+
+    # Functions of class
+    def getOrderTotal(self):
+        total = 0
+        for order in self.orderItems:
+            total += order.getTotalPrice()
+        return total
+
+    def getTotalPerPerson(self):
+        return self.getOrderTotal()/self.numPeople
+
+    def getOrderItemByName(self, dishName):
+        for order in self.orderItems:
+            if (order.dishName == dishName):
+                return order
+        return None
+
+    def addNewOrderItem(self, orderItem):
+        self.orderItems.append(orderItem)
+
+    def addExistingOrderItem(self, orderItem):
+        oi = self.getOrderItemByName(orderItem.dishName)
```

Comment:
I'd preffer to change `oi` to something more explicit like `ortder_item`. But it's your call.

(ref1)